### PR TITLE
vmui: update information about tsdb usage in cluster version

### PR DESCRIPTION
**Pull request by dmitryk-dk**
This improvement adds explanations and expectations to the number of label values on Cardinality Explorer. This explanation is only visible for cluster installation because only in the cluster version the same label could have different values on each node.
Information was added to the documentation module. 

Screenshot 2023-09-19 at 12 18 02

Related issue: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3070

 Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

**Review by Loori-R — COMMENTED** [review]
a few comments

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx [hunk 1]
```diff
@@ -63,8 +63,39 @@ export const TipCardinalityOfSingle: FC = () => (
   </TipCard>
 );
 
-export const TipCardinalityOfLabel: FC = () => (
+export const TipCardinalityOfLabel: FC<{isCluster: boolean}> = ({ isCluster }) => (
   <TipCard title="Dashboard of a label">
+    {isCluster ?
```
[the remark this answers is not in the document]
Comment (dmitryk-dk) [reply app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx]: removed from this module

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx [hunk 2]
```diff
@@ -63,8 +63,39 @@ export const TipCardinalityOfSingle: FC = () => (
...
+      <p>
+          In&nbsp;
+        <a
```
[the remark this answers is not in the document]
Comment (dmitryk-dk) [reply app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx]: Used Hypelink and update a logic a little bit

**Review by Loori-R — APPROVED** [review]
LGTM!

**Pull request merged**